lib/tasks/youth_birthday_reminder.rake

```diff
@@ -1,8 +1,34 @@
 desc "Create a notification for volunteers when a youth has a birthday coming in the next month, scheduled for the 15th of each month in Heroku Scheduler"
 task youth_birthday_reminder: :environment do
   CasaCase.birthday_next_month.each do |casa_case|
-    YouthBirthdayNotification
+    YouthBirthdayNotifier
       .with(casa_case: casa_case)
       .deliver(Volunteer.find_by(id: casa_case.case_assignments.first.volunteer_id))
   end
 end
+
+desc "Create a emancipation_checklist_reminder_notifier notification"
+task emancipation_checklist_reminder_notifier: :environment do
+  CasaCase.birthday_next_month.each do |casa_case|
+    EmancipationChecklistReminderNotifier
+      .with(casa_case: casa_case)
+      .deliver(Volunteer.find_by(id: casa_case.case_assignments.first.volunteer_id))
+  end
+end
+
+desc "Create a followup_notifier notification"
+task followup_notifier: :environment do
+  followup = Followup.all.first
+  deliver_from = User.all.to_a.last
+
+  FollowupNotifier
+    .with(followup: followup, created_by: deliver_from)
+    .deliver(User.find_by(id: 1))
+end
+
+desc "Create a reimbursement_complete_notifier notification"
+task reimbursement_complete_notifier: :environment do
+  ReimbursementCompleteNotifier
+    .with(case_contact: CaseContact.all.first)
+    .deliver(User.find_by(id: 1))
+end
```

Comment:
Not sure I totally understood these. These notifications were used just for your local testing right? For now lets move to a folder like `lib/tasks/development/`.

From what I can tell we don't have any notifications being created in our seeds so those need to be added and then we can get rid of these. None of that is relevant to this PR and I will open another issue to deal with that later.